beam-migrate/beam-migrate.cabal

```diff
@@ -62,7 +62,7 @@ library
   build-depends:       base                 >=4.9     && <5.0,
                        beam-core            >=0.9     && <0.10,
                        text                 >=1.2     && <1.3,
-                       aeson                >=0.11    && <1.6,
+                       aeson                >=0.11    && <1.6 || >=2.0.1.0 && <2.1,
```

Comment:
Why is 2.0.0.0 not allowed? My understanding is the vulnerability applies to all previous versions.

Reply:
The press release said it was fixed in 2.0.1.0. I know. It's strange.

Reply:
But doesn't it apply to all 1.x versions as well? They use `HashMap` without even an option to switch to another implementation.

Also, I don't think it's the job of libraries to ban insecure downstream dependencies via version constraints. That's the job of the application developer / packaging system. Especially in this case, for many applications it is desirable to use `HashMap` anyway. If a user wants to use 2.0.0.0 I don't think Beam should be making that decision.